zap/tests/selene/mod.rs

```diff
@@ -0,0 +1,55 @@
+mod checker;
+
+use std::{borrow::Cow, fs, sync::LazyLock};
+
+use checker::{initialise_selene, Selene};
+use insta::{assert_debug_snapshot, glob, Settings};
+
+static SELENE: LazyLock<Selene> = LazyLock::new(initialise_selene);
```

Comment:
^ This bumps the MSRV of zap to `1.80.0` which is fine as we compile on `stable` and we aren't a rust crate! But good to be aware of :)